app/components/HomePage/CompanyValues.tsx

```diff
@@ -21,7 +20,7 @@ import ContentWrapper from "../ContentWrapper";
 // VALUES COMPONENT DEFINITION
 export default function CompanyValues(props: PageHomeBlocksValues) {
   return (
-    <Box position="relative">
+    <Box position="relative" overflow="hidden">
```

Comment:
Per NextJS docs, "Alternatively, object-fit: "cover" will cause the image to fill the entire container and be cropped to preserve aspect ratio. For this to look correct, the overflow: "hidden" style should be assigned to the parent element." That takes care of the box overflowing